tcconfig-schema/src/main/xsd/terracotta.xsd

```diff
@@ -166,7 +178,24 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
     <xs:attribute name="value" type="xs:string" use="required"/>
   </xs:complexType>
-  
+
+  <xs:complexType name="failover-priority">
+    <xs:choice>
+      <xs:element name="consistency" type="tc:consistency" minOccurs="0"/>
```

Comment:
What's the point of `minOccurs` here?

Reply:
That one was unintentional. Will remove.

Reply:
That one was unintentional. Will remove.